docs/manage/users-remote.rst

```diff
@@ -160,3 +162,11 @@ You can also manage user groups.
 
 To find out how to manage remote users via the WebUI, including adding a new remote user, visit
 :ref:`managing RBAC via the WebUI <manage-users-groups-webui>`.
+
+*****************************
+ Handling SSO Session Expiry
+*****************************
+
+SSO users are redirected to the SSO provider's authentication URIs when their session token has
```

Comment:
Yep exactly! This is different from the `always_redirect` option, which enables this behavior even in the event that a user doesn't have a session token which bypasses the determined sign in page entirely for most cases; the exception being when a user signs out through the WebUI.

If multiple SSO providers have `always_redirect` enabled, each will be attempted to be used.